specification/repository.md

```diff
@@ -78,20 +78,25 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 
 - MUST have a [CHANGELOG.md](templates/CHANGELOG.md) updated for every release
   - The `CHANGELOG.md` is intended to be consumed by humans, and not machines.
+  - The following sub-sections MAY be used, as appropriate or specified:
```

Comment:
I know that you only reordering things. Consider to use https://keepachangelog.com/en/1.0.0/

It is already applied both in https://github.com/signalfx/splunk-otel-dotnet/blob/main/CHANGELOG.md and https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/CHANGELOG.md

Reply:
See https://github.com/signalfx/gdi-specification/pull/45

Reply:
@pellared You suggest this thread can be marked as solved? Or the opposite? 😅